### DISCUSSION TITLE
Fedora dependency

### DISCUSSION THREAD

**Discussion by rimko** [sezanzeb/input-remapper#156] [General]
Just to help Fedora user and future me, on a fresh install of Fedora 34 gnome edition you have install evdev first:
`
sudo dnf install python3-evdev
`
Them you can follow the pip installation procedure:
```
sudo pip install git+https://github.com/sezanzeb/key-mapper.git
sudo systemctl enable key-mapper
sudo systemctl restart key-mapper
```

**Comment by cybermaus** [comment]
I may as well add to the thread, as I too want to be able to find this back:

I could not install on Raspberry OS. evdev was already there for other projects, but I had to `sudo apt-get install gettext` to avoid a `error: [Errno 2] No such file or directory: 'msgfmt': 'msgfmt' ` on the build step.

**Reply by sezanzeb** [reply]
weird, gettext is already a dependency of the deb file: https://github.com/sezanzeb/key-mapper/blob/main/DEBIAN/control

**Reply by cybermaus** [reply]
Well, it needed it. The good news is, it seems to be working on Raspberry Pi. Simple key mappings work already. Now to figure out more complex stuff. There does not seem to be much in the way of manuals/wiki, but I'll try to figure it out before I start asking  many questions.

**Reply by sezanzeb** [reply]
have you seen https://github.com/sezanzeb/key-mapper/blob/main/readme/usage.md and https://github.com/sezanzeb/key-mapper/blob/main/readme/macros.md ?

**Reply by cybermaus** [reply]
BTW: I just realised: My gettext problem was on the build of the deb file, your dependency is on the install of it.

**Comment by ZtereoHYPE** [comment]
Note: the repo and systemd services are now called `input-remapper`
